src/script/components/CellsGlobalView/CellsGlobalView.tsx

```diff
@@ -40,17 +39,15 @@ interface CellsGlobalViewProps {
 
 export const CellsGlobalView = ({cellsRepository = container.resolve(CellsRepository)}: CellsGlobalViewProps) => {
   const {files, status: filesStatus, clearAll, removeFile} = useCellsStore();
-  const {refresh} = useGetAllCellsFiles({cellsRepository});
-  const {
-    searchValue,
-    searchResults,
-    status: searchStatus,
-    handleSearch,
-    handleClearSearch,
-  } = useSearchCellsFiles({
+
+  const {searchValue, handleSearch, handleClearSearch} = useSearchCellsFiles({
     cellsRepository,
   });
 
+  useEffect(() => {
+    void handleSearch('');
+  }, [cellsRepository]);
```

Comment:
this `cellsRepository` dependency shouldn't be there IMO

Reply:
:+